src/hooks/useTwinMakerClient.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+import { configureSdkWithDataSource, DataSourceParams } from 'panels/sdkInit';
+import IoTTwinMaker from 'aws-sdk/clients/iottwinmaker';
+
+export const useTwinMakerClient = (
```

Comment:
Is the TwinMaker client actually used anywhere? What's the purpose of this?

Reply:
@haweston is this from the previous iteration?

Reply:
Alarm panel was using this, but isn't now.

Reply:
It still calls the hook, but no longer uses it in a meaningful way other than making sure the data source is connected.

Reply:
Let's avoid using the UxSDK altogether here if we can, so we don't use different credentials on the browser. Can probably just check if the datasource is configured - just check the `dataSourceParams`

Reply:
Removed